Info on custom Docker images or a binary for GraphQL engine (#4828)
hasura/graphql-engine: docs/graphql/manual/deployment/custom-docker-images.rst

.. meta::
   :description: Requesting custom Docker images for GraphQL engine
   :keywords: hasura, docker, custom images, linux

.. _custom_docker_image:

Custom Docker images or binaries for Hasura GraphQL engine
==========================================================

.. contents:: Table of contents
   :backlinks: none
   :depth: 1
   :local:

If you need a binary for Hasura GraphQL engine to run in a non-Docker environment or you require

> custom Docker images for reasons such as, the default base image *(alpine)* is not compatible with your security scanning tool

Reviewer: ```suggestion
custom Docker images, we can make them available via a support plan. A possible use case could be that the default base image *(alpine)* is not compatible with your security scanning tool
```

Reviewer: This sentence was a bit long, so I split it into two sentences.

> (`Anchore <https://anchore.com/>`_, `Sysdig Secure <https://sysdig.com/products/kubernetes-security/image-scanning/>`_, etc.),

Reviewer: ```suggestion
(`Anchore <https://anchore.com/>`_, `Sysdig Secure <https://sysdig.com/products/kubernetes-security/image-scanning/>`_, etc.).
```

> we can make them available via a support plan.

Reviewer: ```suggestion

```

> To set up this support plan, please contact us using the form on our

Reviewer: ```suggestion
To set up such a support plan, please contact us using the form on our
```

> `contact us page <https://hasura.io/contact-us/>`_ and mention the Linux flavor

Reviewer: ```suggestion
`contact page <https://hasura.io/contact-us/>`_ and make sure to include the Linux flavor
```
